recipes/skytools/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "skytools" %}
+{% set version = "0.0.1b5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/1cosmologist/skytools/archive/v{{ version }}.tar.gz
+  sha256: 5aa3e4d8854885723d2ae43044da1dada8023de807e0accd3dfc8e8693aefb1f
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: true    # [win or py<39]
```

Comment:
What is the reason this is not noarch?

Reply:
One dependency: healpy, is not available on windows. So the win64 recipe failed. Skip was set to true, to fix that failure.

Reply:
This is fine. The recipe can be noarch then and we can merge with failing Windows ci. On the feedstock, it will run on Linux only and the package won't ve installable with the correct error message on Windows.

Reply:
Okay, that is good to know. Changed to noarch. All dependencies have version restrictions now. Will fail win64 CI.